src/pod-security-policy/seccomp/src.rego

```diff
@@ -188,3 +152,18 @@ input_containers[container.name] = container {
 input_containers[container.name] = container {
     container := input.review.object.spec.ephemeralContainers[_]
 }
+
+canonicalize_seccomp_profile(profile) = out {
+    profile.type == "RuntimeDefault"
+    out := "runtime/default"
```

Comment:
Per existing code, this could also count as `docker/default`

Reply:
Would it be better to translate from annotation style to SC style then, since `docker/default` and `runtime/default` both can be mapped to `RuntimeDefault`?

Reply:
But this wouldn't work for `Localhost` type.

Reply:
Had to implement a workaround in translation logic. But I think now the policy should be good.